doc/examples/configuration_tester.py

```diff
@@ -14,4 +14,4 @@
 
 layout.apply()
 
-print layout.generate_fstab()
```

Comment:
Suggestion: add `from __future__ import print_function` so as not to break pylint reporting on py2 (not that it matters much)

Reply:
That's something I had thought of. I'll go ahead and add that in